themes/classic/_dev/js/components/top-menu.js

```diff
@@ -28,38 +28,39 @@ import DropDown from './drop-down';
 export default class TopMenu extends DropDown {
   init() {
     let elmtClass;
-    let self = this;
-    this.el.find('li').hover((e) => {
+    const self = this;
+    this.el.find('li').hover(e => {
       if (this.el.parent().hasClass('mobile')) {
         return;
       }
       const currentTargetClass = $(e.currentTarget).attr('class');
       if (elmtClass !== currentTargetClass) {
-        elmtClass = currentTargetClass;
-        
+        const classesSelected = Array.prototype.slice.call(e.currentTarget.classList).map(e => (typeof e === 'string' ? `.${e}` : false));
+
+        elmtClass = classesSelected.join('');
+
         if (elmtClass && $(e.target).data('depth') === 0) {
-          $(`.${elmtClass} .js-sub-menu`).css({
-            top: $(`.${elmtClass}`).height() + $(`.${elmtClass}`).position().top
+          $(`${elmtClass} .js-sub-menu`).css({
+            top: $(`${elmtClass}`).height() + $(`${elmtClass}`).position().top
```

Comment:
Here is the modification, with classesSelected too, others are linting... Definitly I wont think about committing lint before modifications!